app/boards/arm/corneish_zen/widgets/output_status.c

```diff
@@ -113,14 +113,7 @@ static void output_status_update_cb(struct output_status_state state) {
 
 ZMK_DISPLAY_WIDGET_LISTENER(widget_output_status, struct output_status_state,
                             output_status_update_cb, get_state)
-ZMK_SUBSCRIPTION(widget_output_status, zmk_endpoint_selection_changed);
-
-#if IS_ENABLED(CONFIG_USB_DEVICE_STACK)
-ZMK_SUBSCRIPTION(widget_output_status, zmk_usb_conn_state_changed);
-#endif
-#if defined(CONFIG_ZMK_BLE)
-ZMK_SUBSCRIPTION(widget_output_status, zmk_ble_active_profile_changed);
-#endif
+ZMK_SUBSCRIPTION(widget_output_status, zmk_endpoint_changed);
```

Comment:
Does the endpoint change event actually fire when the active profile becomes cleared? That's one edge case that is handled because the "active profile changed" actually fires when the active profile goes from paired to open/cleared.

Reply:
Unsure. It does check for changes based on active profile changed events (https://github.com/zmkfirmware/zmk/pull/1803/files#diff-5f5f09e0a8d09090e60302308d29a5906d8a41a643992a1d5c4a047033c65688R331), but I haven't checked whether that results in a change to the selected endpoint and therefore fires this event. Will check that.

Reply:
Looks like it doesn't update if you aren't plugged in to USB and the active profile disconnects. Will fix.

Reply:
Added the BLE event handler back in with a note as to why it's still needed.